docker-compose-dev.yaml

```diff
@@ -83,18 +83,35 @@ services:
       - KEYCLOAK_ISSUER=http://localhost/keycloak/realms/Gamify-IT
       - KEYCLOAK_URL=http://keycloak/keycloak/realms/Gamify-IT
 
+  regexgame-backend:
+    container_name: regexgame-backend
+    image: ghcr.io/gamify-it/regexgame-backend:development
+    restart: always
+    pull_policy: always
+    expose:
+      - "80"
+    depends_on:
+      - regexgame-db
+    environment:
+      - POSTGRES_USER=postgres
+      - POSTGRES_PASSWORD=postgres
+      - POSTGRES_URL=postgresql://regexgame-db:5432/postgres
+      - OVERWORLD_URL=http://overworld-backend/api/v1
+      - KEYCLOAK_ISSUER=http://localhost/keycloak/realms/Gamify-IT
+      - KEYCLOAK_URL=http://keycloak/keycloak/realms/Gamify-IT
+
```

Comment:
```suggestion
```
this file is supposed to only run the dependencies so that one can run the backend itself on the host in order to attach a debugger

Reply:
Is it possible that we can meet online and discuss this?

Reply:
Sure

Reply:
Do you have time tonight around 8:00 pm for a Discord chat?

Reply:
Alternatively, tomorrow morning around 10:00 am.